# Type: fix
# Subtype: I1
# English Summary: Cancel the toString() method call on b
# Chinese Summary: 取消b的toString()方法调用
# Entry Number: 29

# === Git Diff Content ===

diff --git a/OsmAnd/src/net/osmand/plus/OsmandApplication.java b/OsmAnd/src/net/osmand/plus/OsmandApplication.java
index f2707bf417..25dd830476 100644
--- a/OsmAnd/src/net/osmand/plus/OsmandApplication.java
+++ b/OsmAnd/src/net/osmand/plus/OsmandApplication.java
@@ -583,7 +583,7 @@ public class OsmandApplication extends Application implements ClientContext {
 		uiHandler.post(new Runnable() {
 			@Override
 			public void run() {
-				AccessibleToast.makeText(uiContext, b.toString(), Toast.LENGTH_LONG).show();
+				AccessibleToast.makeText(uiContext, b, Toast.LENGTH_LONG).show();
 			}
 		});
 	}
